.gitmodules

```diff
@@ -78,4 +78,11 @@
 [submodule "src/BLAKE3"]
 	path = src/BLAKE3
 	url = https://github.com/BLAKE3-team/BLAKE3.git
-
+[submodule "src/boost_redis"]
+	path = src/boost_redis
+	url = https://github.com/boostorg/redis.git
```

Comment:
a bad rebase added this back after it was removed in https://github.com/ceph/ceph/pull/58428. i've opened https://github.com/ceph/ceph/pull/58971 to remove it again